tests/framework/installer/ceph_manifests.go

```diff
@@ -798,7 +798,7 @@ rules:
   verbs: [ "get", "list", "watch", "create", "update", "delete" ]
 ---
 # Aspects of ceph-mgr that require access to the system namespace
-kind: Role
+kind: ClusterRole
```

Comment:
Since this is now a `ClusterRole`, it needs to move out of the `GetClusterRoles()` and into the `GetRookOperator()` method. Otherwise, when multiple clusters are created with the same operator, the ClusterRole will fail to be created the second time.

Reply:
In addition, the ClusterRoleBinding for `rook-ceph-mgr-cluster` needs to move to the other method.